managedtenants/bundles/index_builder.py

```diff
@@ -26,7 +26,11 @@ def __init__(
         )
 
     def build_and_push(
-        self, bundles, hash_string=get_short_hash(), skip_validation=False
+        self,
```

Comment:
nitpick: refactoring the block and adding new params in the same commit makes review harder.

Reply:
noted, thanks for the feedback :)